tools/clang_format_wrapper.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""Wrapper script for `git-clang-format` to avoid race conditions when
+merging/rebasing CPP files with merge conflicts."""
+from collections import namedtuple
+from os import environ as env
+from subprocess import check_output, CalledProcessError
+from typing import List
+import argparse
+import logging
+
+logging.basicConfig(format="[%(levelname)s] %(message)s", level=logging.DEBUG)
```

Comment:
not for myself: reset to logging.INFO before merging?

Reply:
Please put this into the `if __name__ == "__main__"` part (see below).